tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
+    NS.gluster.definitions.save()
+    NS.gluster.config.save()
     NS.publisher_id = "gluster_integration"
+    NS.tendrl_context = \
```

Comment:
In accordance to above comment, if cluster specific `tendrl_context` is loaded, same should be available under NS.tendrl_context, so that all the places we refer `NS.tendrl_context.integration_id` works seamlessly.